### DISCUSSION TITLE
[Bug]: Can't restore backup after version update

### DISCUSSION THREAD

**Discussion by kekegsm** [mbentley/docker-omada-controller#457] [Help]
### Controller Version

v5.13.30.8

### Describe the Bug

I try to restore configuration (*.cfg) file after updated the version, but failed. This error came last december.
I can restore backups from 2023 november.
Same error on every mbentley Omada Controller, all running on Synology NAS.

### Expected Behavior

I can restore backups

### Steps to Reproduce

1. Backup config
2. Update controller version
3. restore config

### How You're Launching the Container

```plain
From container manager
```

### Container Logs

```plain
.
```

**Comment by mbentley** [comment]
1. I can't help without any detail.
2. If you are having problems restoring a cfg backup from the software, it's a software problem, not a containerization problem.  Use the [TP-Link community forums](https://community.tp-link.com/en/business/forum/582) to get help with software problems as I have no access to the source code of the controller and I don't work for TP-Links.

**Reply by kekegsm** [reply]
I tought it is a common problem. We have 14 controller on Synology/Container Manager, and 8 other on Debian Linux PC, and only the containers affected from this problem. And all goes bad last december.
Maybe something like this: https://github.com/mbentley/docker-omada-controller/issues/383

**Reply by mbentley** [reply]
Do you have any logs?  Exactly what it does when you try to restore?  It was previously an issue but it was fixed by #404 at the time.
